managed/ui/src/components/config/PublicCloud/ProviderConfigurationContainer.js

```diff
@@ -52,6 +53,7 @@ const mapDispatchToProps = (dispatch) => {
           dispatch(fetchCloudMetadata());
           const providerUUID = response.payload.data.uuid;
           dispatch(bootstrapProvider(providerUUID, regionFormVals)).then((boostrapResponse) => {
+            toast.success("Successfully created AWS Provider!!")
```

Comment:
change to `'` pls and no need in second `!`

Reply:
Done